packages/cpp/ArmoniK.Api.Common/header/utils/GuuId.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file GuuId.h
+ * @brief Header file for the GuuId class, providing a UUID generation method.
+ */
+
+#pragma once
+
+#include <iomanip>
+#include <iostream>
+#include <random>
+
+/**
+ * @brief The armonik::api::common::utils namespace provides utility classes and functions for the Armonik API.
+ */
+namespace armonik::api::common::utils {
+/**
+ * @class GuuId
+ * @brief The GuuId class provides a static method for generating UUIDs.
+ */
+class GuuId {
+public:
+  /**
+   * @brief Generates a random UUID string.
+   *
+   * This method generates a random UUID string, following the format "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx".
+   *
+   * @return A std::string containing the generated UUID.
+   */
+  static std::string generate_uuid() {
```

Comment:
Good enough for now, but it's not the actual algorithm https://datatracker.ietf.org/doc/html/rfc4122#section-4.4

Reply:
I agree, it's only used in tests for now and will be removed after.

Reply:
Work for another PR